spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -328,7 +331,7 @@ The JSON node is named `<metric-name>`. The JSON leafs are named `<key>[';'<tag-
     "p99": 5608694.0,
     "p999": 5608694.0,
     "count;servlet=two": 29382,
-    "elapsedTime;servlet=two": 25608694,
+    "sum;servlet=two": 25608694,
```

Comment:
I'd suggest that this also stay as elapsedTime for consistency with simpleTimer -- see the conversation from https://github.com/eclipse/microprofile-metrics/pull/621#issuecomment-719547650